drivers/gpio/Kconfig.b91

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2021 Telink Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+# Telink B91 GPIO configuration options
+
+config GPIO_B91
```

Comment:
Probably good to rename this to GPIO_TELINK_B91

Reply:
Done